internal/cmd/app.go

```diff
@@ -79,14 +80,15 @@ func newApp(newRegistry internal.NewRegistry) *cli.App {
 			return nil
 		},
 		Action: func(c *cli.Context) error {
-			car := car{
-				registry:    newRegistry(c.Context, domain, path),
-				verbose:     c.Bool(flagVerbose),
-				veryVerbose: c.Bool(flagVeryVerbose),
-				out:         c.App.Writer,
-			}
+			car := carutil.New(
```

Comment:
ack the arg arity is a bit messy, but we can clean that up once we add "extract" because it is will show all the dimensions of it.